.changelog/unreleased/features/ibc-relayer-cli/2431-show-counterparty.md

```diff
@@ -0,0 +1,2 @@
+- Adds the show-counterparty flag to show every path in a pretty way
```

Comment:
```suggestion
- Add a `--show-counterparty` flag to `hermes query channels` to show every channel, port, chain id and their counterparty's in a pretty format
```

Reply:
Resolved ! I also noticed I entered the wrong issue and corrected it